plugins/server-stats/api/parts/stats.js

```diff
@@ -241,6 +241,35 @@ function fetchDatapoints(db, filter, options, callback) {
             return callback(toReturn);
         }
 
+        if (options.monthlyBreakdown) {
```

Comment:
Please add the  params monthlyBreakdown and license_hosting  to jsdoc definition of the method

Reply:
Updated